core/models/templates.go

```diff
@@ -91,10 +93,10 @@ func loadTemplates(ctx context.Context, db *sql.DB, orgID OrgID) ([]assets.Templ
 const sqlSelectTemplatesByOrg = `
 SELECT ROW_TO_JSON(r) FROM (
      SELECT t.uuid, t.name, (SELECT ARRAY_TO_JSON(ARRAY_AGG(ROW_TO_JSON(tr))) FROM (
-         SELECT tr.namespace, tr.locale, tr.external_locale, tr.external_id, tr.components, JSON_BUILD_OBJECT('uuid', c.uuid, 'name', c.name) as channel
+         SELECT tr.namespace, tr.locale, tr.external_locale, tr.external_id, tr.components, JSON_BUILD_OBJECT('uuid', c.uuid, 'name', c.name) as channel, (CASE WHEN tr.status = 'A' THEN TRUE ELSE FALSE END) AS approved
            FROM templates_templatetranslation tr
            JOIN channels_channel c ON tr.channel_id = c.id
-          WHERE tr.is_active = TRUE AND tr.status = 'A' AND tr.template_id = t.id AND c.is_active = TRUE
```

Comment:
No more filtering out by status as we need to use the approved method for those not approved instead of sending the message without using the template and still fail because we are outside the 24h window